### PR TITLE
Remove helm values placeholder from sample azure yamls

### PR DESCRIPTION
**Pull request by bobotimi**
This PR is based on https://github.com/sonatype/nxrm3-preview-ha-repository/tree/main/nxrm-azure-ha-helm/templates . It removes the helm placeholders from the YAMLs

### CODE REVIEW

**Review by lisadurant — APPROVED** [review]

File: sample-azure-ha-yamls/azure-ha-docker-service.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
```
Comment (mpiggott) [inline sample-azure-ha-yamls/azure-ha-docker-service.yaml:1]: Missing copyright header

File: sample-azure-ha-yamls/azure-ha-ingress-for-docker-connector.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+apiVersion: networking.k8s.io/v1
```
Comment (mpiggott) [inline sample-azure-ha-yamls/azure-ha-ingress-for-docker-connector.yaml:1]: Missing copyright header

File: sample-azure-ha-yamls/azure-ha-docker-service.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  labels:
+    app: nxrm
+spec:
+  type: NodePort
+  selector:
+    app: nxrm
+  ports:
+    - name: docker-service
+      protocol: TCP
+      port: 9090
```
Comment (mpiggott) [inline sample-azure-ha-yamls/azure-ha-docker-service.yaml:15]: Whats port 9090 for?

File: sample-azure-ha-yamls/azure-ha-kustomize-statefulset.yaml [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+        securityContext:
+          runAsUser: 200
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 8081
+        env:
+        - name: DB_NAME
+          value: <db-name>
+       -  name: DB_PORT
+          value: 3306
```
Comment (mpiggott) [inline sample-azure-ha-yamls/azure-ha-kustomize-statefulset.yaml:59]: Is this port right? It isn't the typical postgresql port

File: sample-azure-ha-yamls/azure-ha-logback-tasklogfile-override.yaml [hunk 5]
```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
```
Comment (mpiggott) [inline sample-azure-ha-yamls/azure-ha-logback-tasklogfile-override.yaml:1]: Missing license header

File: sample-azure-ha-yamls/azure-ha-namespaces.yaml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```
Comment (mpiggott) [inline sample-azure-ha-yamls/azure-ha-namespaces.yaml:1]: Missing license header

File: sample-azure-ha-yamls/azure-ha-secret.yaml [hunk 7]
```diff
@@ -0,0 +1,45 @@
+apiVersion: secrets-store.csi.x-k8s.io/v1
```
Comment (mpiggott) [inline sample-azure-ha-yamls/azure-ha-secret.yaml:1]: Missing license header

File: sample-azure-ha-yamls/azure-ha-services.yaml [hunk 8]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```
Comment (mpiggott) [inline sample-azure-ha-yamls/azure-ha-services.yaml:1]: Missing license header

File: sample-azure-ha-yamls/azure-ha-docker-service.yaml (hunk 1 above)
Comment (bobotimi) [reply sample-azure-ha-yamls/azure-ha-docker-service.yaml:10]: Done

File: sample-azure-ha-yamls/azure-ha-ingress-for-docker-connector.yaml (hunk 2 above)
Comment (bobotimi) [reply sample-azure-ha-yamls/azure-ha-ingress-for-docker-connector.yaml:10]: Done

File: sample-azure-ha-yamls/azure-ha-docker-service.yaml (hunk 3 above)
Comment (bobotimi) [reply sample-azure-ha-yamls/azure-ha-docker-service.yaml:24]: It's the port to forward traffic to if you set up a separate ingress for docker.  We've recommended using docker sub domain or reverse proxy if you have more than one docker repository

File: sample-azure-ha-yamls/azure-ha-ingress.yaml [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+kind: Ingress
+metadata:
+  namespace: nxrm
+  name: nxrm-nexus-ingress
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/ssl-redirect: "false"
+spec:
+  rules:
+    - host: example.com #comment out if not using https for load balancer
```
[the remark this answers is not in the document]
Comment (bobotimi) [reply sample-azure-ha-yamls/azure-ha-ingress.yaml:16]: This is in the non-helm yamls so they need to edit it anyway

File: sample-azure-ha-yamls/azure-ha-logback-tasklogfile-override.yaml (hunk 5 above)
Comment (bobotimi) [reply sample-azure-ha-yamls/azure-ha-logback-tasklogfile-override.yaml:6]: Done

File: sample-azure-ha-yamls/azure-ha-secret.yaml (hunk 7 above)
Comment (bobotimi) [reply sample-azure-ha-yamls/azure-ha-secret.yaml:6]: Done

File: sample-azure-ha-yamls/azure-ha-kustomize-statefulset.yaml [hunk 10]
```diff
@@ -0,0 +1,129 @@
...
+            secretKeyRef:
+              name: db-secret
+              key: db-port             
+        - name: NEXUS_SECURITY_INITIAL_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: nxrm-secret
+              key: nxrm-admin-user
+   - name: INSTALL4J_ADD_VM_PARAMS
+          value: "-Xms2703m -Xmx2703m -XX:MaxDirectMemorySize=2703m -Dnexus.licenseFile=/nxrm-license/nxrm-license.lic \
```
[the remark this answers is not in the document]
Comment (bobotimi) [reply sample-azure-ha-yamls/azure-ha-kustomize-statefulset.yaml]: Done

File: sample-azure-ha-yamls/azure-ha-kustomize-statefulset.yaml (hunk 4 above)
Comment (bobotimi) [reply sample-azure-ha-yamls/azure-ha-kustomize-statefulset.yaml]: Set to 5432

File: sample-azure-ha-yamls/azure-ha-ingress-for-docker-connector.yaml [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+  namespace: nxrm
+  name: nxrm-docker-ingress
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/ssl-redirect: "false"
+spec:
+  rules:
+    - host: example.com #comment out if not using https for load balancer
+      http:
+    paths:
```
[the remark this answers is not in the document]
Comment (bobotimi) [reply sample-azure-ha-yamls/azure-ha-ingress-for-docker-connector.yaml]: Done

**Review by oleksiirudyk — APPROVED** [review]

**Pull request merged**